pkgs/development/python-modules/weasyprint/default.nix

```
@@ -45,7 +45,10 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ cairosvg pyphen cffi cssselect lxml html5lib tinycss pygobject2 ];
 
+  # pango.patch can be removed after next release of weasyprint, see:
+  # https://github.com/Kozea/WeasyPrint/issues/1333#issuecomment-818062970
   patches = [
+    ./pango.patch
```

Can we use `fetchpatch` here instead?

```suggestion
    (fetchpatch {
      # https://github.com/Kozea/WeasyPrint/issues/1333#issuecomment-818062970
      url = "https://github.com/Kozea/WeasyPrint/commit/47043a1fd7e50a892b9836466f521df85d597c44.patch";
      sha256 = "0000000000000000000000000000000000000000000000000000000000000000";
    })
```

---

Changed.